JASP-Engine/JASP/R/ttestbayesianindependentsamples.R

```diff
@@ -1022,6 +998,16 @@ TTestBayesianIndependentSamples <- function(dataset=NULL, options, perform="run"
 					plottingError[rowNo] <- state$plottingError[index]
 					
 				} else {
+				    
+				    errors <- .hasErrors(dataset, perform, message = 'short', type = c('observations', 'variance', 'infinity'),
+				                         all.target = variable,
+				                         observations.amount = '< 2')
+				    
+				    if (!identical(errors, FALSE)) {
+				        errorMessage <- errors$message
+				    } else {
+				        errorMessage <- NULL
+				    }
 					
 					result <- try (silent=FALSE, expr= {
```

Comment:
If we find errors we do not want to run this at all.
Additionally, we need the catch structure for exceptions.